src/Helper/Fields/Field_Html.php

```diff
@@ -58,11 +58,15 @@ public function __construct( $field, $entry, Helper_Abstract_Form $gform, Helper
 	 */
 	public function form_data() {
 
-		$data = [];
-		$html = $this->value();
-
-		$data['html'][]                      = $html;
-		$data['html_id'][ $this->field->id ] = $html;
+		$data  = [];
```

Comment:
@jestonihpi upon thinking about this, I'd rather we didn't modify the `Field_Html.php` file. Let's just merge in the HTML data from the `html_id` key in the Field_Repeater.php like you are already doing.

Reply:
Copy.